Parse/src/main/java/com/parse/ParsePush.java

```diff
@@ -422,6 +442,14 @@ public void clearExpiration() {
   }
 
   /**
+   * Sets a UNIX epoch timestamp at which this notification should be delivered, in seconds (UTC).
+   * Scheduled time can not be in the past and must be at most two weeks in the future.
+   */
+  public void setPushTime(long time) {
```

Comment:
Yeah seconds are a terrible choice, but we already have `setExpirationTime(long)` and `setExpirationInterval(long)` in seconds, it would be confusing to have this in millis or in Date.
Note sure what to do, maybe we should add a `Date` version for all three methods. What do you think?